corehq/apps/case_importer/templates/case_importer/partials/ko_import_status.html

```diff
@@ -32,13 +32,29 @@
     <!--ko foreach: result.errors-->
     <!--ko if: rows-->
     <li>
-      <p>
-        <strong data-bind="text: title"></strong>
-        <!--ko if: column--> {% trans 'in column' %}
-        <strong data-bind="text: column"></strong><!--/ko-->
-      </p>
+      <!--ko if: column-->
+      <p><strong>
+        <!-- ko if: rows.length == 1-->
+        {% blocktrans %}
+          <span data-bind="text: rows.length"></span> row had an invalid
+          "<span data-bind="text: column"></span>" cell and was not saved
+        {% endblocktrans %}
+        <!--/ko-->
+        <!-- ko if: rows.length > 1-->
+        {% blocktrans %}
+          <span data-bind="text: rows.length"></span> rows had invalid
+          "<span data-bind="text: column"></span>" cells and were not saved
+        {% endblocktrans %}
+        <!--/ko-->
+      </strong></p>
+      <!--/ko-->
+      <!--ko if: !column -->
+      <p><strong data-bind="text: title"></strong></p>
+      <!--/ko-->
       <p data-bind="text: description">
       </p>
+      <p data-bind="text: sample">
```

Comment:
This should probably also use `visible: sample`. I think if there's no sample, it'll still add whitespace due to paragraph margins/padding.